neonvm/config/default-vxlan/manager_config_patch.yaml

```diff
@@ -21,6 +21,13 @@ spec:
         - "--zap-time-encoding=iso8601"
         - "--zap-log-level=info"
         - "--zap-stacktrace-level=panic"
+        # FIXME: Rebase & move this to the appropriate place after merging #779.
+        #
+        # See #775 and its links.
+        # * cache.writeback=off - forces writes to be synchronous (avoid hidden caching)
+        # * cache.direct=on     - forces using direct IO (don't abuse host's page cache!)
+        # * cache.no-flush=on   - ignores disk flush operations (not needed; our disks are ephemeral)
+        - "--qemu-disk-cache-settings=cache.writeback=off,cache.direct=on,cache.no-flush=on"
```

Comment:
why are the CLIs below using different defaults than this here?

Reply:
late reply, but: not quite sure what you mean

Reply:
Ah I see yeah, right now they have that default because (a) it's simpler, and (b) IMO it's a safer fallback than this untested concoction of flags :sweat_smile: